### DISCUSSION TITLE
Second domain as public domain

### DISCUSSION THREAD

**Discussion by t2noob** [simple-login/app#2130] [Self-hosting question]
So I have a SimpleLogin self hosted install that works perfectly. 
I just bought a new domain lastname.com and want to add it so my users (Family) can create new aliases with the new lastname.com domain. 
How can I go ahead an add it? when sending an email I get "Relay Access Denied" when I try to send an email to an alias I created on SimpleLogin. 
I added it to the .env file with the OTHER_ALIAS_DOMAINS variable and when I do the initializing I get the confirmation that lasname.com has already been added to the public domains.
Basically what I did was add all the DNS records a custom domain uses to be able to be tied to the simplelogin app. I guessing it might be a postfix issue, but I have no idea how to add a second domain to that and not break what I already have working. Any help would be appreciated it.
I am on the simplelogin/app-ci:latest which I've been using without issues.
